actionpack/test/controller/render_test.rb

```diff
@@ -481,6 +490,21 @@ def test_implicit_no_content_response
     get :empty_action
     assert_response :no_content
   end
+
+  def test_implicit_render_template_no_respond_to
+    get :json_action, format: 'json'
+    assert_response :success
```

Comment:
Can you put something in the template and assert the content is what you expect?